test/services/searchResultEntity.test.ts

```diff
@@ -0,0 +1,78 @@
+/**
+ * @jest-environment node
+ */
+import httpStatusCode from 'http-status-codes';
+
+import { findById } from '@/services/searchResultEntity';
+import { createUser } from '@/repositories/user';
+import { createMultipleSearchResults } from '@/repositories/searchResult';
+
+import { user } from '../factories/user';
+import { searchResult } from '../factories/searchResult';
+
+describe('findById:', () => {
+  const defaultUUID = 'cb77e671-2447-4197-897d-540bc114df48';
+  describe('given valid id', () => {
```

Comment:
How about give the code a little space to breath 🌊

```suggestion
  const defaultUUID = 'cb77e671-2447-4197-897d-540bc114df48';
  
  describe('given valid id', () => {
```

Reply:
Fixed in https://github.com/nimblehq/ic-nextjs-prabesh-dahal/pull/78/commits/e56350f43b948ee59561e6d3b21aed7291338acd